src/routes/profile-route/profile-sidebar.jsx

```diff
@@ -48,6 +49,19 @@ export const ProfileSidebar = () => {
               </div>
             );
           })}
+          <div className="flex">
```

Comment:
The button works great and I can't wait for it to be merged!! However it doesn't align with the way Josh set up the original buttons in the sidebar.

It would be better to add this button under menuItems, and have it rendered by the map function that's creating the other buttons in the sidebar. It will be easier to maintain changes to the UI later. Under menuItems, you can add a key:value pair for the onClick handler that can be passed to Sidebar.Item. I would also suggest changing the container for the entire sidebar item from a div to a button:

```
 {menuItems.map((item, index) => {
            return (
              <button key={index} className="flex  ">
                <img className="h-8 w-8 " src={item.src} alt={item.title} />
                <div className="mt-1">
                  <Sidebar.Item link={item.link} title={item.title} onClick{item.handleClick}/>
                </div>
              </button>
            );
          })}
```